_includes/footer.html

```diff
@@ -1,33 +1,29 @@
-<footer class="site-footer">
-    <div class="footer-learn-more">
-        <div class="wrapper">
-            <p>Want to try Probo today? <a href="https://app.probo.ci">Get started.</a></p>
-            <div class="right">
-                <a href="https://www.facebook.com/proboci" title="Probo on Facebook" target="_blank"><i class="fa fa-facebook"></i></a>
-                <a href="https://github.com/ProboCI/probo" title="Probo on Github" target="_blank"><i class="fa fa-github-alt"></i></a>
-                <a href="https://twitter.com/ProboCI" title="Sweet Probo Tweets" target="_blank"><i class="fa fa-twitter"></i></a>
+<footer class="footer">
+    <div class="footer__upper">
+        <div class="footer__upper-wrapper">
+            <p class="footer__upper-left">Want to try Probo today? <a href="https://app.probo.ci">Get started.</a></p>
+            <div>
+                <a href="https://www.facebook.com/proboci" class="footer__upper-link" title="Probo on Facebook" target="_blank"><i class="fa fa-facebook" aria-hidden="true"></i></a>
```

Comment:
> You'll need to use CSS to make this link text not visible.

What's the best way to do this? All I can find is that old -9999px hack and other gross stuff. I'm hoping for a magic `text-visibility: accessibility` property or something, but I guess that's a long shot.